Update SSO docs and add GDPR docs (#2701)
coralproject/talk: docs/source/version-5-gdpr.md

---
title: GDPR Compliance

> permalink: v5/integrating/gdpr/

Reviewer: ```suggestion
permalink: /v5/integrating/gdpr/
```

- **Delete my account**: Users can request deletion of their account. Deleted account requests are pending for 24 hours to allow the user to download their comments, or to change their mind and reactivate their account before the expiry. Account deletions remove all of their comments from the site, all their comments and actions from the database, and their account info from our system.
- **Add an email to an OAuth/external account**: Users are prompted to add an email to their non-Coral account (Facebook, Google, external, etc) so that they can take part in GDPR and other features requiring email communication.
- **Change my username**: Users can update their username. This is capped at once every 2 weeks.
- **Change my email**: Users can change their email.
- **Change my password**: Users can change their password.

## GDPR with SSO

As many newsrooms often implement their own [SSO solutions](/talk/v5/integrating/sso/),
we also provide API support to manage GDPR features directly from your own Account or My Profile page.

We provide the following GraphQL mutations designed to allow you to integrate it into your existing user
interfaces or exports.

> - `downloadUser(id: ID!)` - lets you grab the direct link to download a users

Reviewer: ```suggestion
- `requestUserCommentsDownload` - lets you grab the direct link to download a users
```

  account in a zip format. From there, you can integrate it into your existing
  data export or simply proxy it to the user to allow them to download it
  elsewhere in your UI.

> - `delUser(id: ID!)` - lets you delete the specified user

Reviewer: ```suggestion
- `deleteUserAccount` - lets you delete the specified user
```
